### PR TITLE
Created class for hard coded messages and implemented it

### PR DESCRIPTION
**Pull request by Tadas-M**

### CODE REVIEW

File: University-advisor/Forms/ForgotPasswordForm.cs [hunk 1]
```diff
@@ -38,23 +39,23 @@ private void ResetPasswordButton_Click(object sender, EventArgs e)
...
+                    Logger.Log(Messages.passwordUpdateFailed);
+                    MessageBox.Show(Messages.passwordUpdateFailed);
                 }
             }
             else
             {
-                Logger.Log("User attempted to change password");
-                MessageBox.Show("User with provided information does not exist.");
+                Logger.Log(Messages.userAttemptedToChangePassword);
+                MessageBox.Show(Messages.userDontExist);
```
Comment (MartinCepulionis) [inline University-advisor/Forms/ForgotPasswordForm.cs]: Maybe we should add message to Logger that the attempt was not successfull?

File: University-advisor/Services/EmailSenderService.cs [hunk 2]
```diff
@@ -31,11 +32,11 @@ public static void SendEmail(string recipient, string subject, string message)
                 msg.Subject = subject;
                 msg.Body = message;
                 SendThroughSmtp(msg);
-                Logger.Log("Email was successfully sent.");
+                Logger.Log(Messages.sendEmailSuccess);
```
Comment (avizen-j) [inline University-advisor/Services/EmailSenderService.cs:35]: Could you export client set up strings to Constant directory as well? I am talking about 18, 19 lines.

**Review by avizen-j — CHANGES_REQUESTED** [review]
Some more code needs to be extracted to Constants directory.

File: University-advisor/Services/EmailSenderService.cs (hunk 2 above)
Comment (Tadas-M) [reply University-advisor/Services/EmailSenderService.cs:35]: Yeah, sure. good idea

File: University-advisor/Forms/ForgotPasswordForm.cs (hunk 1 above)
Comment (Tadas-M) [reply University-advisor/Forms/ForgotPasswordForm.cs]: Okay, fixed that. Let me know if you have more suggestions

**Pull request merged**